### DISCUSSION TITLE
Specify a tag in the import within a NAV section

### DISCUSSION THREAD

**Discussion by goochusmc** [jdoiro3/mkdocs-multirepo-plugin#69] [General]
Is there any way to specify a tag for the imported portion. I see, and have used the branch option but would like to be able to link together different tags from different repos.

**Comment by goochusmc** [comment]
Nevermind, after looking through the code I see that you should just be able to use the tag name in the branch argument for the import and it should work just fine.

**Reply by jdoiro3** [reply]
Apologies for the late reply. Glad you were able to figure it out. I will update the docs for the plugin to make this clear for future users.

**Reply by jdoiro3** [reply]
I just updated the docs to reflect this. Thanks again for using the plugin and helping improve it!

**Comment by navi771** [comment]
When i installed multirepo plugin using this command "pip install mkdocs-multirepo-plugin" in Window and i configure plugins in .yaml file
eg:
plugins:

search
multirepo:
# (optional) tells multirepo to cleanup the temporary directory after site is built.
cleanup: true
# if set the docs directory will not be removed when importing docs. When using this with a nav section in an imported repo
# you must keep the docs directory in the path (e.g., docs/path/to/file.md).
keep_docs_dir: true
-When i am using "plugins" keyword in yaml then it getting error as below :

ERROR - Config value 'plugins': The "multirepo" plugin is not installed
Aborted with 1 Configuration Errors!

-But when i using "plugin" in yaml then project running but multirepo features not working

SO please help me how we can use in window as steps
